src/bot/index.js

```diff
@@ -40,6 +40,9 @@ bot.messagePipe(
       shoutout,
       project,
       roulette,
+      command('reset', ({ isModerator, isBroadcaster }) =>
+        isModerator || isBroadcaster ? { memory: null, message: 'Ok malord.' } : `Hum.. nop.`,
```

Comment:
To avoid chat spam, we should avoid sending something if we reject

Reply:
But without feedback from the bot we don't know if it refuses to obey (kinda expected) or if it is just buggy

Reply:
Imagine, if we do so. Some user can spam the chat with !reset and the bot will respond every time ?

Reply:
You're right